src/components/quiz/create/Config.jsx

```diff
@@ -32,10 +31,15 @@ class Config extends React.Component {
     }
 
     createRoom = () => {
-        console.log("create room called");
-        const {numSymbols,numRounds} = this.state;
-        var roomID = setUpRoom(numSymbols,numRounds + 1,'');
-        this.props.updatePageType('created',roomID);
+      const Industry = null;
+      const Sector = null;
+      const MarketCaps = null;
+      const {numSymbols,numRounds} = this.state;
+
+      initSymbols(Industry,Sector,MarketCaps,numSymbols).then((symbolsL) => {
+            let roomID = setUpRoom(symbolsL,numRounds,'');
+            this.props.updatePageType('created',roomID);
+      });
```

Comment:
I'm not sure that we agreed on adding this back in the day of the demo... @jack-y-xu ?